### PR TITLE
[incubator/buzzfeed-sso] Add support for multiple paths for the same host

### PR DESCRIPTION
**Pull request by anas-aso**
#### Is this a new chart
no

#### What this PR does / why we need it:
to be able to expose multiple endpoints behind buzzfeed-sso

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Special notes for your reviewer:
~~this PR introduces **breaking changes**~~ The breaking change will be delayed until the next major release.

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

**Comment by anas-aso** [comment]
@darioblanco @komljen can I have you reviews please ?

File: incubator/buzzfeed-sso/templates/ingress.yaml [hunk 1]
```diff
@@ -40,10 +40,12 @@ spec:
     - host: {{ .domain }}
       http:
         paths:
-          - path: {{ .path }}
+          {{- range .paths }}
+          - path: {{ . }}
             backend:
               serviceName: {{ $fullName }}-proxy
               servicePort: http
+          {{- end }}
```
Comment (darioblanco) [inline incubator/buzzfeed-sso/templates/ingress.yaml:54]: Good idea. Let's make this backwards compatible with the current clients and then will be ready to go

**Review by darioblanco — CHANGES_REQUESTED** [review]

File: incubator/buzzfeed-sso/templates/ingress.yaml [hunk 2]
```diff
@@ -40,10 +40,18 @@ spec:
     - host: {{ .domain }}
       http:
         paths:
+          {{- if .path }}
```
Comment (anas-aso) [inline incubator/buzzfeed-sso/templates/ingress.yaml:43]: this is required since we set `.path` as deprecated.

**Review by darioblanco — APPROVED** [review]
LGTM

**Pull request merged**